core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.ts

```diff
@@ -58,7 +82,89 @@ export class DotEmaComponent implements OnInit {
         });
     }
 
+    ngAfterViewInit(): void {
+        this.document.defaultView?.addEventListener('message', (event: MessageEvent) => {
+            // This should be the host the user uses for nextjs, because this can trigger react dev tools messages
+            if (event.origin !== this.host) {
+                return;
+            }
+
+            this.handlePostMessage(event.data)();
+        });
+    }
+
+    /**
+     * Create the url to edit a contentlet
+     *
+     * @private
+     * @param {string} inode
+     * @return {*}
+     * @memberof DotEmaComponent
+     */
+    private createEditContentletUrl(inode: string): string {
+        return `/c/portal/layout?p_p_id=content&p_p_action=1&p_p_state=maximized&p_p_mode=view&_content_struts_action=%2Fext%2Fcontentlet%2Fedit_contentlet&_content_cmd=edit&inode=${inode}`;
+    }
+
+    /**
+     * Handle the iframe load event
+     *
+     * @private
+     * @param {CustomEvent} event
+     * @memberof DotEmaComponent
+     */
+    onIframeLoad(_: Event) {
+        this.dialogIframe.nativeElement.contentWindow?.removeEventListener(
+            'ng-event',
+            this.handleNgEvent.bind(this)
+        );
+
+        this.dialogIframe.nativeElement.contentWindow?.document.addEventListener(
```

Comment:
Does this means that we are (from the backend) sending events to the windows and to the document?

Reply:
Made this by mistake, fixed.